.editorconfig

```diff
@@ -0,0 +1,8 @@
+[*]
```

Comment:
I am thinking adding this will also be another file to upkeep. Eslint should be good enough.

Reply:
Trust me when I say this: we won't be touching this file ever. It's just to give VS Code something to do than manually doing it.